lib/moon_web/examples/design/snackbar_example/auto_close.ex

```diff
@@ -0,0 +1,59 @@
+defmodule MoonWeb.Examples.Design.SnackbarExample.AutoClose do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+
+  use MoonWeb, :example
+
+  alias Moon.Design.Snackbar
+  alias Moon.Design.Button
+
+  def render(assigns) do
+    ~F"""
+    <Snackbar id="snackbar_16" timeout={1000}>
```

Comment:
Nit: better to extract those repeated timeout values out to common variables and avoid using magic numbers.

Reply:
it's not a magic number, it's just a milliseconds in example. shown directly for transparency